examples/combobox/combobox-autocomplete-both.html

```diff
@@ -51,16 +51,16 @@ <h2 id="ex_label">Example</h2>
         <div class="combobox combobox-list">
           <div class="group">
             <input id="cb1-input" class="cb_edit" type="text" role="combobox" aria-autocomplete="both"
-              aria-expanded="false" aria-controls="lb1">
-            <button type="button" id="cb1-button" aria-label="Open" tabindex="-1">
+              aria-expanded="false" aria-controls="cb1-listbox">
+            <button type="button" id="cb1-button" aria-label="State" aria-expanded="false" aria-controls="cb1-listbox" tabindex="-1">
```

Comment:
The name of the listbox is "States" plural, and this button controls that, so I recommend:

```suggestion
            <button type="button" id="cb1-button" aria-label="States" aria-expanded="false" aria-controls="cb1-listbox" tabindex="-1">
```

Reply:
Fixed